doc/mvp-guideline-production.md

```
@@ -78,83 +140,69 @@ docker exec webhook-app-prod /app/main fetch-historical \
 ```bash
-make start-leaderboardscoring-app-prod
+docker exec shared-redis redis-cli KEYS "leaderboard:*:all_time"
 ```
 
-Check added redis keys:
-```bash
-docker exec rankr-shared-redis redis-cli KEYS "leaderboard:*:all_time"
-```
 Output:
-```log
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)